hello-world-example/README.md

```diff
@@ -76,7 +76,7 @@ cfy secrets create openstack_auth_url --secret-string <value>
 For **AWS**:
 
 ```shell
-cfy install aws.yaml -i aws_region_name=eu-central-1
```

Comment:
Maybe provide both examples. One as a way to use default and one as a way to override.